README.md

```diff
@@ -92,6 +92,52 @@ gpg --keyserver hkp://pool.sks-keyservers.net --recv-keys <last eight symbols of
 gpg --keyserver hkp://keyserver.ubuntu.com --send-keys <last eight symbols of gnupg keyId>
 ```
 
+### Stencil Server API
+
+For serving the protobuf descriptor set artifacts and their versions we use a Stencil Server.
+This also helps us to easily update the descriptor sets by allowing us to push Protobuf Descriptor sets directly.
+
+#### Endpoints
+
+```http
+GET https://stencil-hostname.example.com/artifactory/proto-descriptors/:stencil_repo/:version
+PUT https://stencil-hostname.example.com/artifactory/proto-descriptors/:stencil_repo/:version
+GET https://stencil-hostname.example.com/metadata/proto-descriptors/:stencil_repo/version
+PUT https://stencil-hostname.example.com/metadata/proto-descriptors/:stencil_repo/version
+```
+All above endpoints are behind HTTP Basic Auth.
```

Comment:
`The section avoids handling/recommending AUTH mechanism as it is left to the user. The samples below assume that basic Auth set up.`
WDYT?

Reply:
These docs are about a specific implementation as of now.